modules/ecs6-class/line.js

```diff
@@ -2,17 +2,29 @@ const Point = require("./point");
 
 class Line {
     constructor({ point1 = new Point(), point2 = new Point(), n = undefined, slope = undefined }) {
+        if(typeof(point1)!="object" && point1 !=undefined || typeof(point2)!="object" && point2 !=undefined){
+            throw new Error('argument must be type point')
```

Comment:
maybe I have point1 = {id:1} , or point1 = [1,2,3]
I want to know exactly which parameter threw an error